src/DataTable/filters/DropdownFilter.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Form,
+} from '../..';
+import { newId } from '../../utils';
+
+const DEFAULT_VALUE = 'dropdown-filter-default';
```

Comment:
would an empty string be better here?

Reply:
I worry that an empty string would match with zero if you're using inexact equality. Which you might want to do because of numbers getting turned into strings in the DOM.

Reply:
You're using `===` below and 0 should be converted to "0" right?  Setting a custom string to mean 'nothing' seems like misusing the value attribute to me.